### ISSUE TITLE
RBAC

### ISSUE THREAD

**Issue by imbaniac** [Alonski/CoinPanion#23]
- Should we have different roles for supporter and creator?
- Should we have an admin role?
- Should we not show some pages for unauthorized users except Dashboard

**Comment by Alonski** [comment]
About different roles: I think everyone can be both a supporter and a creator. No need to split them and I think will make our job harder.

Admin Role: Might need it in the future. Maybe in the end if we have time.

Users without an injected web3 instance (no address):

- Should be redirected to the About Page?
- Can't access Dashboard?

**Comment by Josephrp** [comment]
good idea to redirect to About page (where we explain what Coinpanion is and how to join)
